### DISCUSSION TITLE
v0.1-alpha-07 is out!

### DISCUSSION THREAD

**Discussion by Arxance** [Arxance/Devamped#3] [Announcements]
This update includes all mob effect icons and few changes. Also the first update that bundles both JE and BE pack!

<--- Devamped v0.1-alpha-07 (July 21, 2023) --->
> [BE] 385 Items | 121 Retextured GUI | 5 Blocks
> [JE] 294 Items | 36 Retextured GUI | 5 Blocks
[+] Meat food related got changed
[+] Added All mob effect icons on both JE and BE
[+] Added proper potions for JE port

<hr /><em>This discussion was created from the release <a href='https://github.com/PixelAlphas/Devamped/releases/tag/'>v0.1-alpha-07 is out!</a>.</em>

**Comment by Ziziiiiiiiiii** [comment]
🔥🔥
